services/things/persistence/src/main/java/org/eclipse/ditto/services/things/persistence/actors/strategies/events/AbstractThingEventStrategy.java

```diff
@@ -46,9 +46,12 @@ protected AbstractThingEventStrategy() {
     @Override
     public Thing handle(final T event, @Nullable final Thing thing, final long revision) {
         if (null != thing) {
+            final String issuedAt = event.getDittoHeaders().get("_issuedAt");
+
             ThingBuilder.FromCopy thingBuilder = thing.toBuilder()
                     .setRevision(revision)
                     .setModified(event.getTimestamp().orElse(null));
+                    // .setMetadata(event.getResourcePath(), event.getEntity());
```

Comment:
I think these changes can be removed, right?
We would need a mechanism to generically add metadata provided in the headers (with a defined prefix) to the thing's `Metadata`.

Reply:
Yes, you are right, thats from me "playing around".